pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -96,8 +96,22 @@ type Domain struct {
 }
 
 type DomainStatus struct {
-	Status LifeCycle
-	Reason StateChangeReason
+	Status     LifeCycle
+	Reason     StateChangeReason
+	Interfaces []InterfaceStatus
+}
+
+type InterfaceStatus struct {
+	Name string
+	Mac  string
+	Ip   string
+	// TODO: should this be a simple []string?
```

Comment:
Can you check what format Kubernetes uses, maybe in Endpoint objects? And you can also check in multus, they put an annotation with assigned addresses on Pod object.

Reply:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#endpoints-v1-core
Subsets: [ { Addresses: [{"ip": "10.10.1.1"}, {"ip": "10.10.2.2"}]

So the difference is between:
IPs: ["1.1.1.1/24", "2.2.2.2/1", ...] and Addresses: [{"ip": "10.10.1.1"}, {"ip": "10.10.2.2"}]

Reply:
For Pod's only the IP is reported (no subnet) (https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#podstatus-v1-core). Followng that partternm, this should be a list of strings (IPs).

Thus as it is right now.